akka-http2-support/src/main/scala/akka/http/impl/engine/http2/framing/Http2FrameParsing.scala

```diff
@@ -117,8 +117,9 @@ private[http2] class Http2FrameParsing(shouldReadPreface: Boolean) extends ByteS
             // TODO: check frame size
             // TODO: check flags
             // TODO: check reserved flag
```

Comment:
You can remove this as well. The "reserved flag" is actually the sign when we read a complete `readIntBE`. So, technically the error message should then also be "Reserved flag in WINDOW_UPDATE" must be 0.